### PR TITLE
ICDS changes

### PR DESCRIPTION
**Pull request by lwyszomi**
@emord 
https://manage.dimagi.com/default.asp?265770
https://manage.dimagi.com/default.asp?265450
https://manage.dimagi.com/default.asp?264815
https://manage.dimagi.com/default.asp?265769
https://manage.dimagi.com/default.asp?265164

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -1261,13 +1261,13 @@ def get(self, request, *args, **kwargs):
...
         config = {
             'month': tuple(test_date.timetuple())[:3],
-            'aggregation_level': 1,
         }
         location = request.GET.get('location_id', '')
         config.update(get_location_filter(location, self.kwargs['domain']))
         loc_level = get_location_level(config.get('aggregation_level'))
 
         data = []
+        config['aggregation_level'] = 5
```
Comment (emord) [inline custom/icds_reports/views.py]: Why this change?

File: custom/icds_reports/tests/test_export_data.py [hunk 2]
```diff
@@ -964,91 +964,91 @@ def test_awc_infrastructure_export(self):
...
+                            "Percentage AWCs reported weighing scale: mother and child"
                         ],
                         [
                             "st1",
-                            "50.00 %",
-                            "25.00 %",
-                            "30.77 %",
-                            "38.46 %",
-                            "13.46 %"
+                            "2600.00 %",
```
Comment (emord) [inline custom/icds_reports/tests/test_export_data.py]: Why are these numbers so high?

File: custom/icds_reports/views.py (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/views.py]: @emord Currently in denominator we must sum all records (passing the filters) where ```infra_last_update_date == null```, so we can't take data for specific location level (we will get max 1), we must get records for all awcs in selected location.

File: custom/icds_reports/tests/test_export_data.py (hunk 2 above)
Comment (lwyszomi) [reply custom/icds_reports/tests/test_export_data.py]: @emord I checked in output data for tests and a lot of records have field ```infa_last_update_date != null``` so we have a situation where nominator > denominator.

File: custom/icds_reports/views.py (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/views.py]: @emord yes, even if a user is looking for the state level, data would be aggregate based on AWC data.

**Comment by lwyszomi** [comment]
@emord updated
